### ISSUE TITLE
Listen: problem downloading files on desktop

### ISSUE THREAD

**Issue by soapdog** [Himalayan-Academy/Siva-Siva-App#282]
The Desktop Version:

No files are going to the card "listen-my-audio"  I call on property inspector on the fld "AudioList" it's text is empty.

The Mobile Version (because I have produced a standalone to my iPhone)

We ARE getting files that are downloaded, in card "listen-my-audio" field "AudioList"

_Originally posted by @Brahmanathaswami in https://github.com/Himalayan-Academy/Siva-Siva-App/issues/274#issuecomment-813465923_

**Comment by soapdog** [comment]
I can't reproduce this. On my desktop it is working fine.

**Comment by Brahmanathaswami** [comment]
There was a problem with an older 

modified:   models/jnanam/jnanam.sqlite.gz

We got  it from devhap, we have failed to get SivaSivaApp documents with on update. But this worked 

SivaSivaApp "document" 

<img width="824" alt="image" src="https://user-images.githubusercontent.com/3675722/115132505-acef2380-9f9c-11eb-8c07-4455a7950a8c.png">

Why the issue went away? I can't tell you, but it is gone :  I can't reproduce this. On my desktop it is working fine.   And on mobile card "listen-my-audio" field "AudioList" are being listed, working  fine 

Case closed
